src/site/content/en/mini-apps/mini-app-conclusion/index.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: Conclusion
```

Comment:
Also not going to yield strong SEO results. "Concluding thoughts about mini apps from a web developer" or something like that would probably perform better

Reply:
Went with your suggestion!